src/styles/brackets_fonts.less

```diff
@@ -27,57 +27,53 @@
 @font-weight-light: 200;
 
 
-/* SourceSansPro Regular */
+/* @sansFontFamily Regular */
 @font-face {
-    font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-Regular.ttf');
+    font-family: @sansFontFamily;
     font-weight: normal;
     font-style: normal;
 }
 
 
-/* SourceSansPro Light */
+/* @sansFontFamily Light */
 @font-face {
-    font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-Light.ttf');
+    font-family: @sansFontFamily;
     font-weight: @font-weight-light;
     font-style: normal;
 }
 
 
-/* SourceSansPro Semibold */
+/* @sansFontFamily Semibold */
 @font-face {
-    font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-Semibold.ttf');
+    font-family: @sansFontFamily;
     font-weight: @font-weight-semibold;
     font-style: normal;
 }
 
-/* SourceSansPro Italic */
+/* @sansFontFamily Italic */
 @font-face {
-    font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-It.ttf');
+    font-family: @sansFontFamily;
     font-weight: normal;
     font-style: italic;
 }
 
 
-/* SourceCodePro Regular */
+/* @sourceFontFamily Regular */
 @font-face {
-    font-family: 'SourceCodePro';
-    src: url('fonts/SourceCodePro/SourceCodePro-Regular.ttf');
+    font-family: @sourceFontFamily;
     font-weight: normal;
     font-style: normal;
 }
 
-/* SourceCodePro Medium */
+/* @sourceFontFamily-medium */
 @font-face {
-    font-family: 'SourceCodePro-Medium';
-    src: url('fonts/SourceCodePro/SourceCodePro-Medium.ttf');
+    font-family: @sourceFontFamily-medium;
     font-weight: normal;
     font-style: normal;
 }
 
 /* Font Stacks */
 
-@sansFontFamily: 'SourceSansPro', Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
+@sansFontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
+@sourceFontFamily: "Menlo Regular", Consolas, Inconsolata, "Vera Sans", "Lucida Console", Courier, fixed;
+@sourceFontFamily-medium: "Menlo Regular", Consolas, Inconsolata, "Vera Sans", "Lucida Console", Courier, fixed;
```

Comment:
I'm not sure about this one, it probably needs to happen in another commit on my upstream PR.